api/src/main/java/io/minio/MinioClient.java

```diff
@@ -3263,12 +3263,17 @@ public boolean bucketExists(String bucketName)
    * @throws RegionConflictException thrown to indicate passed region conflict with default region.
    * @throws XmlParserException thrown to indicate XML parsing error.
    */
+  @Deprecated
   public void makeBucket(String bucketName)
       throws ErrorResponseException, IllegalArgumentException, InsufficientDataException,
           InternalException, InvalidBucketNameException, InvalidKeyException,
           InvalidResponseException, IOException, NoSuchAlgorithmException, RegionConflictException,
           XmlParserException {
-    this.makeBucket(bucketName, null, false);
+
+    MakeBucketArgs.Builder makeBucketArgs = new MakeBucketArgs.Builder();
+    makeBucketArgs.bucket(bucketName);
+    MakeBucketArgs args = makeBucketArgs.build();
+    this.makeBucket(args);
```

Comment:
You could use `this.makeBucket(MakeBucketArgs.newBuilder().bucket(bucketName).build())` directly everywhere.